src/olympia/reviewers/utils.py

```diff
@@ -1017,6 +1018,19 @@ def log_action(
         kwargs = {'user': user or self.user, 'created': timestamp, 'details': details}
         self.log_entry = ActivityLog.objects.create(action, *args, **kwargs)
 
+        attachment = None
+        if self.data.get('attachment_file') and self.data.get('attachment_input'):
+            raise RuntimeError('Cannot upload both a file and input.')
```

Comment:
We don't need this extra validation here - the form is already validating the inputs. (If we did, we typically just use `assert xxx`)